content/300-guides/050-database/890-neon.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Using Prisma with Neon'
+metaTitle: 'Using Prisma with Neon'
+metaDescription: 'Guide to using Prisma with Neon'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+This guide explains how to connect Prisma to Neon, establish connections when using Prisma Client in serverless functions, and resolve [connection timeout](#connection-timeouts) issues.
```

Comment:
> establish connections when using Prisma Client in serverless functions

I don't see how this page relates to that. Maybe better to drop that part of the sentence?

Reply:
I think this is in reference to the connection-pooled option Neon offers which is more valuable in serverless environments. I'll adjust the section to be more clear